components/invoice_ninja/actions/create-client/create-client.mjs

```diff
@@ -0,0 +1,246 @@
+    },
+    customValue4: {
+      type: "string",
+      label: "Custom Value 4",
+      description: "A custom field for storing additional information",
+      optional: true,
+    },
+    vatNumber: {
+      type: "string",
+      label: "VAT Number",
+      description: "The client's VAT (Value Added Tax) number, if applicable",
+      optional: true,
+    },
+    idNumber: {
+      type: "string",
+      label: "ID Number",
+      description: "A unique identification number for the client, such as a tax ID or business registration number",
+      optional: true,
+    },
+    number: {
+      type: "string",
+      label: "Number",
+      description: "A system-assigned unique number for the client, typically used for invoicing purposes",
+      optional: true,
+    },
+    shippingAddress1: {
+      type: "string",
+      label: "Shipping Address 1",
+      description: "First line of the client's shipping address",
+      optional: true,
+    },
+    shippingAddress2: {
+      type: "string",
+      label: "Shipping Address 2",
+      description: "Second line of the client's shipping address, if needed",
+      optional: true,
+    },
+    shippingCity: {
+      type: "string",
+      label: "Shipping City",
+      description: "City of the client's shipping address",
+      optional: true,
+    },
+    shippingState: {
+      type: "string",
+      label: "Shipping State",
+      description: "State of the client's shipping address",
+      optional: true,
+    },
+    shippingPostalCode: {
+      type: "string",
+      label: "Shipping Postal Code",
+      description: "Postal code of the client's shipping address",
+      optional: true,
+    },
+    shippingCountryId: {
+      propDefinition: [
+        app,
+        "countryId",
+      ],
+      label: "Shipping Country ID",
+      description: "The ID of the country for the client's shipping address",
+      optional: true,
+    },
+    groupSettingsId: {
+      propDefinition: [
+        app,
+        "groupSettingsId",
+      ],
+      optional: true,
+    },
+    isTaxExempt: {
+      type: "boolean",
+      label: "Is Tax Exempt",
+      description: "Flag which defines if the client is exempt from taxes",
+      optional: true,
+    },
+    hasValidVatNumber: {
+      type: "boolean",
+      label: "Has Valid VAT Number",
+      description: "Flag which defines if the client has a valid VAT number",
+      optional: true,
+    },
+    classification: {
+      type: "string",
+      label: "Classification",
+      description: "Classification of the client",
+      options: CLASSIFICATION_OPTIONS,
+      optional: true,
+    },
+    settings: {
+      type: "object",
+      label: "Settings",
+      description: "An array of settings objects in JSON format. **Example: {\"currency_id\": 1, \"timezone_id\": 5, \"date_format_id\": 1, \"language_id\": 1}** [See the documentation](https://api-docs.invoicing.co/#tag/clients/POST/api/v1/clients) for further details",
```

Comment:
I think `settings` takes an object instead of an array.
```suggestion
      description: "A settings object in JSON format. **Example: {\"currency_id\": 1, \"timezone_id\": 5, \"date_format_id\": 1, \"language_id\": 1}** [See the documentation](https://api-docs.invoicing.co/#tag/clients/POST/api/v1/clients) for further details",
```